Tests/SQLiteTests/SQLite3Tests.swift

```diff
@@ -63,6 +63,17 @@ class SQLite3Tests: XCTestCase {
             if let results = try database.execute("SELECT * FROM `foo`").first {
                 XCTAssertEqual(results.data["bar"], unicode)
             }
+            else {
```

Comment:
Hi ! thanks for the fix, and the tests as well. About Coverage, would it work to have something like : 
``` swift
            let results = try database.execute("SELECT * FROM foo WHERE bar = 42").first
            XCTAssertNotNil(results)
            XCTAssertEqual(results?.data["bar"], "42")
            XCTAssertEqual(results?.data["baz"], "Life")
```
so that every line gets executed in the tests ?